### ISSUE TITLE
Serving static files from a separate domain

### ISSUE THREAD

**Issue by koorchik** [jhthorsen/mojolicious-plugin-assetpack#12]
Hello, @jhthorsen.The plugin is great and it will be nice to have one more useful feature in it. Serving static files from a separate domain is a common practice and I often use it in large projects. Briefly the idea is described in Google best practices - (https://developers.google.com/speed/docs/best-practices/request#ServeFromCookielessDomain).

Separate domain(s) for static files has following advantages: 
- no cookies send on each request. This is especially useful when you use Mojolicious sessions as they are stored in cookies and clients send whole session with every request.
- More request done in parallel. Browsers have limits for sending parallel request to one domain. So, with separate domain static can be loaded in parallel.
- Serve files directly (by absolute url) from CDN (or Amazon S3).

As I see you already have pull-request ( #4 ) for the feature . I guess "url_prefix"(or something like it) option solves the problem.
